packages/client/src/components/Actions/ActionsList.js

```diff
@@ -53,63 +54,66 @@ function ActionsList({
 
   if (!actions.length) {
     ActionComponents.push(
-      <div
-        className="column is-full p-0 is-flex is-justify-content-center is-align-items-center has-text-grey"
-        style={{ minHeight: 175 }}
-        key="no-transactions"
-      >
-        No pending transactions
+      <div className="flex-1 mx-0 p-3 has-text-centered">
+        You do not have any pending actions.
       </div>
     );
   } else {
+    ActionComponents.push(
+      <div className="px-5 m-0 columns has-text-left has-text-grey is-align-items-center is-hidden-touch">
+        <div className="column px-0 is-1">#</div>
+        <div className="column px-0 is-4">Info</div>
+        <div className="column is-3">Status</div>
+        <div className="column px-0 is-2">Signatures</div>
+        <div className="column px-0 is-2">Action</div>
+      </div>
```

Comment:
can we make this a `table` and use the markup structure from https://github.com/DapperCollectives/VESSEL/blob/main/packages/client/src/components/TransactionTable.js#L73-L93 ?

since both of these are on the same page they should be nearly identical in spacing/mobile 

![image](https://user-images.githubusercontent.com/15314629/192843219-c4a2b33b-451c-4e52-8304-62c8e92924c2.png)

![image](https://user-images.githubusercontent.com/15314629/192843290-a5673336-28be-4945-8322-28ee5eba63a9.png)

Reply:
In case of mobile - confirm button should be below, is that possible to achieve with tables?

Reply:
@KsenijaZivkovic might not be - you might just have to make the desktop confirm `is-hidden-mobile` and then have a "mobile" confirm button that is just a full table row only visible on mobile